### PR TITLE
Galaxia mainpage with static pages 

### PR DESCRIPTION
**Pull request by stutisk**
Added Static MainPage with  Aside Bar , Navbar with search bar and avatar , Notes and added React Router 
image
image

### CODE REVIEW

File: src/pages/Home.module.css [hunk 1]
```diff
@@ -1,14 +1,58 @@
...
-  }
-  .btn:hover {
-    background: var(--red-colord2);
-  }
\ No newline at end of file
+  border-radius: var(--border-radius1);
+  color: var(--secondry-color);
+  font-size: var(--size-1);
+  background: var(--red-colorl1);
+  padding: 10px 20px 10px 20px;
```
Comment (Hamdrive) [inline src/pages/Home.module.css]: you can use shorthand to implement vertical and horizontal padding accordingly
`padding: 10px 20px`
also do use rems for px greater than 10-16

File: src/pages/Home.module.css [hunk 2]
```diff
@@ -1,14 +1,58 @@
...
+}
+.aside {
+  width: 18rem;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  min-height: auto;
+
+  letter-spacing: 0.01785714em;
+  font-family: "Google Sans", Roboto, Arial, sans-serif;
```
Comment (Hamdrive) [inline src/pages/Home.module.css]: If this font family is global, then you can apply it to either root or reset CSS instead

**Review by Hamdrive — COMMENTED** [review]
Overall very neat code and good implementations done
Have noted some suggestions do have a look
Also very nicely designed notes app!

**Review by abhij1607 — COMMENTED** [review]
Very Clean Aesthetic code | | Modular files | | should look at hamza's suggestions

also, suggest taking a look at why the checks failed

**Comment by stutisk** [comment]
made the suggested changes

**Pull request merged**